src/host/getset.cpp

```diff
@@ -374,6 +374,16 @@ void ApiRoutines::GetNumberOfConsoleMouseButtonsImpl(ULONG& buttons) noexcept
             // ECHO on with LINE off is invalid.
             RETURN_HR_IF(E_INVALIDARG, WI_IsFlagSet(mode, ENABLE_ECHO_INPUT) && WI_IsFlagClear(mode, ENABLE_LINE_INPUT));
         }
+        if (WI_IsFlagSet(mode, ENABLE_MOUSE_INPUT))
+        {
+            gci.GetActiveInputBuffer()->PassThroughEnableButtonEventMouseMode(true);
+            gci.GetActiveInputBuffer()->PassThroughEnableSGRExtendedMouseMode(true);

```

Comment:
Only calling these 2 seems to make far manager work, but I am not sure if we should be sending other VT sequences through as well/in different cases (like only button events versus all mouse events), since the console mode only has 1 umbrella `ENABLE_MOUSE_INPUT` mode and not more specific ones as far as I know

Reply:
Can you try if Win32 Vim works? The lacking of mouse support in Win32 Vim is the root cause of several issues. It also drives me to NeoVim personally.

Reply:
We probably want 1003 instead of 1002 -- that's the "any event" mode which includes hover (the win32 event model automatically reports hover!)